File: code-samples/voice/call-queues.java
import com.ringcentral.*;
import com.ringcentral.definitions.*;

public class Update_CallQueue_Members {
	static RestClient rcsdk;
	public static void main(String[] args) {
		var obj = new Update_CallQueue_Members();
		rcsdk = new RestClient("client_id", "client_secret", "server_url");
		try {
			rcsdk.authorize("username", "extension_number", "password");
			obj.add_callqueue_members();
		} catch (RestException | IOException e) {
			e.printStackTrace();
		}
	}
	public void add_callqueue_members() throws RestException, IOException {
		var resp = restClient.restapi().account().callqueues().get();
		for (var group : resp.records) {
			if (group.name.equals("Sales team")) {
				var parameters = new CallQueueBulkAssignResource();
				parameters.addedExtensionIds = new String[] {"888888888", "999999999"};
				restClient.restapi().account().callqueues(group.id).bulkassign().post(parameters);
				System.out.println("Members added.");
				break;
			}
		}
	}
}

File: code-samples/voice/call-queues.cs
using System;
using System.Threading.Tasks;
using RingCentral;

namespace Update_CallQueue_Members
{
  class Program
  {
    static RestClient restClient;
    static async Task Main(string[] args)
    {
      restClient = new RestClient(
      Environment.GetEnvironmentVariable("RC_APP_CLIENT_ID"),
      Environment.GetEnvironmentVariable("RC_APP_CLIENT_SECRET"),
      Environment.GetEnvironmentVariable("RC_SERVER_URL"));
      await restClient.Authorize(Environment.GetEnvironmentVariable("RC_USER_JWT"));

      await add_callqueue_members();
    }
    static private async Task add_callqueue_members()
    {
      var resp = await restClient.Restapi().Account().CallQueues().Get();
      foreach (var group in resp.records)
      {
        if (group.name == "Support Department")
        {
          var parameters = new CallQueueBulkAssignResource();
          parameters.addedExtensionIds = new string[] { "888888888", "999999999" };
          await restClient.Restapi().Account().CallQueues(group.id).BulkAssign().Post(parameters);
          Console.WriteLine("Members added");
          break;
        }
      }
    }
  }
}
